library/src/main/kotlin/app/futured/donut/DonutProgressView.kt

```diff
@@ -14,13 +15,9 @@ import androidx.annotation.ColorInt
 import androidx.core.animation.doOnEnd
 import androidx.core.content.ContextCompat
 import androidx.core.content.res.use
+import app.futured.donut.extensions.hasDuplicatesBy
 import app.futured.donut.extensions.sumByFloat
 
-/*
-Ideas:
-- tooling in layout editor (testing data)
-- turn on / off corner path effect with configurable number of sides
- */
 class DonutProgressView @JvmOverloads constructor(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3317dfecd010d9a5776f0a2a2e33736acb814042/library/src/main/kotlin/app/futured/donut/DonutProgressView.kt#L21"></span>Class DonutProgressView is too large. Consider splitting it into smaller pieces.</td>
    </tr>
  </tbody>
</table>